include/openssl/ssl.h

```diff
@@ -2082,8 +2090,8 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client);
  * May return a negative integer if an error occurs.
  * A connection should be aborted if the SCTs are deemed insufficient.
  */
-typedef int(*ssl_ct_validation_cb)(const CT_POLICY_EVAL_CTX *ctx,
-                                   const STACK_OF(SCT) *scts, void *arg);
+typedef int (*ssl_ct_validation_cb) (const CT_POLICY_EVAL_CTX * ctx,
+                                     const STACK_OF(SCT) *scts, void *arg);
```

Comment:
It looks like we prefer to keep the ```)(``` nestled together for the function (type) arguments.
Also, no space in ```*ctx```.